src/Widget/Options/Input.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Form\Widget\Options;
+
+/**
+ * Form input options
+ */
+trait Input
+{
+    public function autofocus(bool $value = true): self
+    {
+        $new = clone $this;
+        $new->options['autofocus'] = $value;
```

Comment:
There's no guarantee that this trait will be attached to a class that also has Common trait so `$this->options` may be unavailable.